README.md

```diff
@@ -1,3 +1,19 @@
+
+<p style="text-align:center;" align="center"><a href="https://layer5.io/meshery"><img align="center" style="margin-bottom:20px;" src="https://raw.githubusercontent.com/layer5io/layer5/master/assets/images/meshery/meshery-logo-tag-light-text-side.png"  width="70%" /></a><br /><br /></p>
+
+# learn-layer5
+
+[![Docker Pulls](https://img.shields.io/docker/pulls/layer5/meshery-nsx-sm.svg)](https://hub.docker.com/r/layer5/meshery-nsx-sm)
+[![Go Report Card](https://goreportcard.com/badge/github.com/layer5io/meshery-nsx-sm)](https://goreportcard.com/report/github.com/layer5io/meshery-nsx-sm)
```

Comment:
Thanks, @gunjanmimo. don't forget to update these references

- https://hub.docker.com/r/layer5/learn-layer5
- https://github.com/layer5io/learn-layer5